src/main/java/javaClasses/homework_4/davit_balabekyan/math/Calculations.java

```diff
@@ -0,0 +1,23 @@
+package javaClasses.homework_4.davit_balabekyan.math;
+
+public class Calculations {
+
+    public static int getMean(int firstIntNumber, int secondIntNumber, int thirdIntNumber) {
```

Comment:
Davit jan pls change the method logic, to accept varargs, like int ... numbers. In this case your method accepts only 3 arguments, but we want to give any number of arguments within it

Reply:
Please make the same change for the other methods

Reply:
Done) All methods have been changed